net/upnp/src/www/services_upnp.php

```diff
@@ -94,6 +95,12 @@ function miniupnpd_validate_port($port)
             $pconfig[$fieldname] = $config['installedpackages']['miniupnpd']['config'][0][$fieldname];
         }
     }
+
+    // ensure backwards compatibility
+    if (!isset($pconfig['num_permuser'])) {
```

Comment:
best to leave the empty value, no need to persist defaults.

Reply:
OK, I removed that.  I had it in order to prevent people from accidentally removing some of their entries if they change the number but don't get it far up enough past however many entries they already had, but I suppose that's a thing that's unlikely to happen.